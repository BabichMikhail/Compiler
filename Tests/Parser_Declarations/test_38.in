TYPE 
	MyType_1 = integer;
	MyType_2 = MyType_1;
	MyType_3 = array of MyType_2;
	MyType_4 = array of string[50];
var
	Arr : MyType_3;
	Arr_2 : MyType_4 = ('qwe', 'rty', 'asd', 'fgh', 'zxc', 'vbn');