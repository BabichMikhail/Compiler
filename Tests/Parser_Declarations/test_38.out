Type   MyType_1
   Type   integer

Type   MyType_2
   Type   integer

Type   MyType_3
   DynArray   
   Type   integer

Type   MyType_4
   Array   1   6
   string   50

Var   Arr
   DynArray   
   Type   integer

Var   Arr_2
   Array   1   6
   string   50
   qwe
   rty
   asd
   fgh
   zxc
   vbn

